README.rst

```diff
@@ -1,101 +0,0 @@
-**********************************************************
```

Comment:
where and why README.rst is gone?

Reply:
changed but forgot to put it back - done now